components/dashboard/src/settings/SelectAccountModal.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+ import { SelectAccountPayload } from "@gitpod/gitpod-protocol/lib/auth";
+ import { useEffect, useState } from "react";
+ import { gitpodHostUrl } from "../service/service";
+ import Modal from "../components/Modal";
+ import SelectableCard from "../components/SelectableCard";
+ import info from '../images/info.svg';
+
+export function SelectAccountModal(props: SelectAccountPayload & {
+    close: () => void
+}) {
+
+    const [useAccount, setUseAccount] = useState<"current" | "other">("current");
+
+    useEffect(() => {
+    }, []);
+
+    const continueWithCurrentAccount = () => {
+        props.close();
+    };
+
+    const continueWithOtherAccount = () => {
+        const accessControlUrl = gitpodHostUrl.asAccessControl().toString();
+
+        const loginUrl = gitpodHostUrl.withApi({
+            pathname: '/login',
+            search: `host=${props.otherUser.authHost}&returnTo=${encodeURIComponent(accessControlUrl)}`
+        }).toString();
+
+        const logoutUrl = gitpodHostUrl.withApi({
+            pathname: "/logout",
+            search: `returnTo=${encodeURIComponent(loginUrl)}`
+        }).toString();
+
+        window.location.href = logoutUrl;
+    };
+
+    return (<Modal visible={true} onClose={props.close}>
+    <h3 className="pb-2">Select Account</h3>
+    <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4">
+        <p className="pb-2 text-gray-500 text-base">You are attempting to authorize a provider that is already connected with your other account on Gitpod.</p>
+
+        <div className="mt-4 flex rounded-md w-full bg-gray-200 p-4 mx-auto">
+            <img className="w-4 h-4 m-1 ml-2 mr-4" src={info} />
+            <span>
+                Disconnect a provider in one of you accounts, if you like to continue with the other account.
+            </span>
+        </div>
+
+        <div className="mt-4 flex-grow flex flex-row justify-around align-center">
+            <SelectableCard className="w-2/5 h-56" title="Current Account" selected={useAccount === 'current'} onClick={() => setUseAccount('current')}>
+                <div className="flex-grow flex flex-col justify-center align-center">
+                    <img className="m-auto rounded-full w-24 h-24" src={props.currentUser.avatarUrl} alt={props.currentUser.name}/>
```

Comment:
**question**: Could we add some top and bottom padding for the account avatar? 😇

Reply:
py-2 or py-4?

Reply:
Let's go with `py-4` to create some visual balance with the rest of the horizontal whitespace.

Reply:
done